scalyr_agent/third_party/tcollector/collectors/0/ifstat.py

```diff
@@ -83,18 +87,19 @@ def main():
         for line in f_netdev:
             # Scalyr edit
             for interface in network_interface_prefixes:
-                m = re.match("\s+(%s%s):(.*)" % (interface, NETWORK_INTERFACE_SUFFIX), line)
+                # 2->TODO is it important to expect at least one space character?
```

Comment:
I don't think it is important to expect at least one.I think using `*` is fine.

Reply:
Got it.